### DISCUSSION TITLE
Next version release date?

### DISCUSSION THREAD

**Discussion by ariaieboy** [caprover/caprover#1776] [General]
Hi @githubsaturn,
The last version of caprover is released about 20+ months ago.
Since then, we had lots of commits on the master branch. 
I want to know when we can have the next version.
Some outdated services like netdata have been updated in master, but it's still outdated "it's uses debian Stretch as base image that is deprecated in June 30, 2022"

**Comment by githubsaturn** [comment]
New version will be released early Aug.
